jcl/src/java.management/share/classes/java/lang/management/MemoryNotificationInfo.java

```diff
@@ -89,9 +89,15 @@
 	 *            {@link #MEMORY_COLLECTION_THRESHOLD_EXCEEDED} notifications,
 	 *            this will be the number of times that the collection usage
 	 *            threshold was passed.
+	 * @throws NullPointerException
+	 *            when poolName or usage is null
 	 */
 	public MemoryNotificationInfo(String poolName, MemoryUsage usage, long count) {
-		super();
+		if (poolName == null)
+			throw new NullPointerException("Null poolName");
```

Comment:
We normally externalize the Strings using the java preprocessor MSG statement. See an example https://github.com/eclipse/openj9/blob/master/jcl/src/java.management/share/classes/java/lang/management/MemoryUsage.java#L93-L94

Add the new messages to https://github.com/eclipse/openj9/blob/master/jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties to find a spare key.